src/stock-operations/stock-operations-table.component.tsx

```diff
@@ -229,14 +247,10 @@ const StockOperations: React.FC<StockOperationsTableProps> = () => {
         </div>
       ),
       actions: (
-        <Tooltip align="bottom" label="Edit Stock Operation">
-          <Button
-            type="button"
-            size="sm"
-            className="submitButton clear-padding-margin"
-            iconDescription={"View"}
-            kind="ghost"
-            renderIcon={Edit}
+        <OverflowMenu flipped={"true"} aria-label="overflow-menu">
+          <OverflowMenuItem itemText="Complete" onClick={handleOnComplete} />
```

Comment:
Also we should display the completion button/item conditionally for only items which does not have completion status. Yo